pkg/apis/garden/v1beta1/defaults.go

```diff
@@ -25,10 +25,12 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 
 // SetDefaults_Shoot sets default values for Shoot objects.
 func SetDefaults_Shoot(obj *Shoot) {
-	if obj.Spec.Backup == nil {
-		obj.Spec.Backup = &Backup{
-			IntervalInSecond: DefaultETCDBackupIntervalSeconds,
-			Maximum:          DefaultETCDBackupMaximum,
```

Comment:
@swapnilgm I think you misunderstood my last comment regarding this section.
You need to handle both cases (`obj.Spec.Backup == nil` because we want to use the default backup config if the user does not specify any backup configuration; and `obj.Spec.Backup != nil` because we have renamed `IntervalInSecond` into `Schedule` and this field won't be present for existing clusters.)